kuma/settings/common.py

```diff
@@ -1209,6 +1209,12 @@ def _get_languages_and_locales():
         ]),
         "JSON array listing tag suggestions for documents"
     ),
+    COMMON_REASONS_TO_BAN_USERS=(
```

Comment:
I strongly believe this should live into [``user contants``](https://github.com/mozilla/kuma/blob/master/kuma/users/constants.py)